### PR TITLE
chore: simpler and new drip scheme naming

### PR DESCRIPTION
**Pull request by manuelpolzhofer**

### CODE REVIEW

File: src/token.sol [hunk 1]
```diff
@@ -18,7 +18,7 @@ struct InputNFTType {
     string ipfsHash;
 }
 
-contract FundingNFT is ERC721, Ownable {
+contract DripToken is ERC721, Ownable {
```
Comment (CodeSandwich) [inline src/token.sol:21]: `DripToken` or `DripsToken`? The whole feature is called `drips` :thinking:

File: src/test/nft.t.sol [hunk 2]
```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.7;
```
Comment (CodeSandwich) [inline src/test/nft.t.sol:2]: `nft.t.sol` -> `token.t.sol`

File: src/token.sol [hunk 3]
```diff
@@ -30,35 +30,35 @@ contract FundingNFT is ERC721, Ownable {
     string public contractURI;
     bool public initialized;
 
-    struct NFTType {
+    struct Type {
```
Comment (CodeSandwich) [inline src/token.sol:33]: `type` is a solidity keyword making it impossible to write `Type storge type`. It's probably not a big deal and hopefully it'll never hit us hard :thinking:

File: src/token.sol (hunk 1 above)
Comment (manuelpolzhofer) [reply src/token.sol:21]: True, `DripsToken` makes more sense

File: src/test/nft.t.sol (hunk 2 above)
Comment (manuelpolzhofer) [reply src/test/nft.t.sol:2]: good catch, fixed

**Review by CodeSandwich — APPROVED** [review]
:+1:

File: src/token.sol (hunk 3 above)
Comment (manuelpolzhofer) [reply src/token.sol:33]: worst case a  `_`

**Pull request merged**